test/hisi_sec_test/test_hisi_sec.c

```diff
@@ -3384,12 +3375,16 @@ static int sec_sva_test(void)
 		}
 		req[i].cb = async_cb;
 		req[i].cb_param = &datas[i];
+
+		datas[i].tid = i;
+		datas[i].req = &req[i];
+		datas[i].setup = &setup[i];
```

Comment:
miss bd_pool?

Reply:
bd_pool is already set by init_bd_pool() function.

thr_data[] are global variable array, and datas[] are internal variable array. A lot of initializations are done in datas array. If we switch to the internal array, we don't need to copy bd_pool to thr_data array any more.